proto/cosmos/slashing/genesis.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+package cosmos.slashing;
+
+option go_package            = "github.com/cosmos/cosmos-sdk/x/slashing/types";
+option (gogoproto.equal_all) = true;
+
+import "gogoproto/gogo.proto";
+import "cosmos/slashing/slashing.proto";
+
+// GenesisState - all slashing state that must be provided at genesis
+message GenesisState {
+  Params params = 1 [
+    (gogoproto.casttype) = "Params",
+    (gogoproto.nullable) = false
+  ];
+
+  repeated SigningInfos signing_infos = 2 [
+    (gogoproto.moretags) = "yaml:\"signing_infos\"",
+    (gogoproto.nullable) = false
+  ];
+
+  repeated ValidatorMissedBlocks missed_blocks = 3 [
+    (gogoproto.moretags) = "yaml:\"missed_blocks\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+// SigningInfos 
+message SigningInfos {
+  string address = 1;
+  ValidatorSigningInfo signing_infos = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"signing_infos\""
```

Comment:
May be we shouldn't change the yaml and json tags. Let's stick to original tags

Reply:
the changing from map to array is a breaking change, and will need a migrate script (which I'm tackling in #6839). Since it's breaking, it would go in breaking the tags too, to make them clearer to future users.